core/dbt/cli/main.py

```diff
@@ -870,6 +869,7 @@ def freshness(ctx, **kwargs):
 @p.project_dir
 @p.select
 @p.selector
+@p.show_output_format
```

Comment:
is this change intentional?

Reply:
ah, I see. it used to be in the `unit_test` command... I think I added it initially to apply to the output format of the rendered diff but I don't think it applies any longer after the daff formatting changes